x/gamm/pool-models/balancer/pool.go

```diff
@@ -17,14 +17,15 @@ import (
 
 //nolint:deadcode
 const (
-	errMsgFormatSharesAmountNotPositive       = "shares amount must be positive, was %d"
-	errMsgFormatTokenAmountNotPositive        = "token amount must be positive, was %d"
-	errMsgFormatTokensLargerThanMax           = "%d resulted tokens is larger than the max amount of %d"
-	errMsgFormatSharesLargerThanMax           = "%d resulted shares is larger than the max amount of %d"
+	nonPostiveSharesAmountErrFormat = "shares amount must be positive, was %d"
+	nonPostiveTokenAmountErrFormat  = "token amount must be positive, was %d"
+	sharesLargerThanMaxErrFormat    = "%d resulted shares is larger than the max amount of %d"
+	invalidInputDenomsErrFormat     = "input denoms must already exist in the pool (%s)"
+
+	// the errors below are being used in export_test
 	errMsgFormatFailedInterimLiquidityUpdate  = "failed to update interim liquidity - pool asset %s does not exist"
 	errMsgFormatRepeatingPoolAssetsNotAllowed = "repeating pool assets not allowed, found %s"
 	errMsgFormatNoPoolAssetFound              = "can't find the PoolAsset (%s)"
```

Comment:
The top messages were changed to be suffixed with `ErrFormat`. Can we change these as well please for consistency?

Reply:
good idea!